utils/notifications.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+import os
+
+URL = "https://api.pushover.net/1/messages.json"
+
+def send_notification(title, message):
```

Comment:
Suggest that you check if API tokens are defined in the environment and only post if they are.

Reply:
Why resolved?

Reply:
Next commit